src/content/faq/11_crashing.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
this faq is slightly misleading because this only covers hard crashes, not plugin related crashes

Reply:
ah alright. it can be axed then. no prob